metadata_tools/util.py

```
@@ -4,12 +4,46 @@
 import os
 import re
 import numpy as np
+import cspyce
+import re
```

Flake8 (7.3.0)

[error] redefinition of unused 're' from line 5

(F811)